app/user/rest.py

```diff
@@ -926,3 +926,39 @@ def send_annual_usage_data(user_id, start_year, end_year, markdown_en, markdown_
     send_notification_to_queue(saved_notification, False, queue=QueueNames.NOTIFY)
 
     return jsonify({}), 204
+
+
+def send_requested_report_ready(report):
```

Comment:
Could you move this function to `app/report/utils.py` and add a unit test for this function?

Reply:
done